builders/build_capo.sh

```diff
@@ -66,14 +68,30 @@ if [ ! -f "$OC_REGISTRY_AUTH_FILE" ]; then
 fi
 
 DEST_IMAGE="quay.io/$USERNAME/origin-release:$TAG"
-FROM_IMAGE="registry.svc.ci.openshift.org/ocp/release:$RELEASE"
+FROM_IMAGE="registry.svc.ci.openshift.org/origin/release:$RELEASE"
```

Comment:
is it worth making this a env variable?

Reply:
perhaps - I'm not exactly clear on why it was changed but that something the devs who use the script will figure out. The script was apparently broken so I do not see any reason not to merge this now and keep iterating over the script.